trade-module/src/main/java/io/edpn/backend/trade/adapter/config/BeanConfig.java

```diff
@@ -0,0 +1,54 @@
+package io.edpn.backend.trade.adapter.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+import io.edpn.backend.util.IdGenerator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.retry.backoff.ExponentialBackOffPolicy;
+import org.springframework.retry.policy.SimpleRetryPolicy;
+import org.springframework.retry.support.RetryTemplate;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Configuration("TradeModuleBeanConfig")
+public class BeanConfig {
+
+
+    @Bean(name = "tradeIdGenerator")
+    public IdGenerator idGenerator() {
+        return new IdGenerator();
+    }
+
+    @Bean(name = "tradeObjectMapper")
+    @Primary
+    public ObjectMapper objectMapper() {
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.registerModule(new Jdk8Module());
+        JavaTimeModule javaTimeModule = new JavaTimeModule();
+        javaTimeModule.addSerializer(LocalDateTime.class, new LocalDateTimeSerializer(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
+        mapper.registerModule(javaTimeModule);
+        return mapper;
```

Comment:
Hmm, just wondering how necessary having module specific object mapper configuration is? And also, how safe, considering we could end up with different configurations on each "end" of de/serialization.